Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -604,38 +597,37 @@ public uint256 LastReceivedBlockHash()
                     .Where(w => w != null)
                     .OrderBy(o => o.LastBlockSyncedHeight)
                     .FirstOrDefault()?.LastBlockSyncedHash;
-            }
 
-            Guard.Assert(lastBlockSyncedHash != null);
-            this.logger.LogTrace("(-):'{0}'", lastBlockSyncedHash);
-            return lastBlockSyncedHash;
+                Guard.Assert(lastBlockSyncedHash != null);
+                this.logger.LogTrace("(-):'{0}'", lastBlockSyncedHash);
+                return lastBlockSyncedHash;
+            }
         }
 
         /// <inheritdoc />
-        public List<UnspentOutputReference> GetSpendableTransactionsInWallet(string walletName, int confirmations = 0)
+        public IEnumerable<UnspentOutputReference> GetSpendableTransactionsInWallet(string walletName, int confirmations = 0)
         {
             Guard.NotEmpty(walletName, nameof(walletName));
             this.logger.LogTrace("({0}:'{1}',{2}:{3})", nameof(walletName), walletName, nameof(confirmations), confirmations);
 
             Wallet wallet = this.GetWalletByName(walletName);
-            List<UnspentOutputReference> res;
+            IEnumerable<UnspentOutputReference> res;
```

Comment:
you can put this inline with the initialization of `res` I guess.

Reply:
Thanks. Fixed.